en-US/win10/samples/PinMappingsDb.md

```diff
@@ -0,0 +1,204 @@
+---
+layout: default
+title: Dragonboard Pin Mappings
+permalink: /en-US/win10/samples/PinMappingsDb.htm
+lang: en-US
+---
+##Dragonboard Pin Mappings
+
```

Comment:
Shouldn't this include a picture of the DB Pinout diagram?
![Dragonboard Pin Header]({{site.baseurl}}/images/PinMappings/DB_Pinout.png)